Sources/DangerSwiftKantoku/Kantoku+IssueComments.swift

```diff
@@ -46,11 +46,11 @@ extension Kantoku {
     }
     
     func post(_ summaries: [PostableIssueSummary], as level: CommentLevel) {
-        
         for summary in summaries {
             let message = summary.issueMessage
             let filePath = summary.documentLocation?.relativePath(against: workingDirectoryPath)
-            
+
+
```

Comment:
[nits]
I guess this diff is just some mistake...?

Reply:
done